src/main/java/androidx/text/Html.kt

```diff
@@ -53,12 +58,12 @@ fun CharSequence.parseAsHtml(flags: Int? = 0): Spanned {
  * not know how to interpret.
  */
 fun CharSequence.parseAsHtml(
-    flags: Int? = 0,
+    @SuppressLint("InlinedApi") flags: Int = FROM_HTML_MODE_LEGACY,
```

Comment:
The reason I did not use `FROM_HTML_MODE_LEGACY ` directly is because it was introduced in API 24: https://developer.android.com/reference/android/text/Html.html#FROM_HTML_MODE_LEGACY.

Reply:
primitive and string constants are inlined by the compiler.

Reply:
Awesome! In that case, thanks for the fix.